charts/argo-workflows/values.yaml

```diff
@@ -29,6 +29,11 @@ fullnameOverride:
 # -- Override the Kubernetes version, which is used to evaluate certain manifests
 kubeVersionOverride: ""
 
+# -- Override APIVersions
+apiVersionOverrides:
+  # -- String to override apiVersion of autoscaling rendered by this helm chart
+  autoscaling: "" # autoscaling/v2
```

Comment:
nits:
We don't need to show the following row on README, 
```
| apiVersionOverrides | object | `{"autoscaling":""}` | Override APIVersions |
```

so maybe like this?
```diff
-# -- Override APIVersions
apiVersionOverrides:
  # -- String to override apiVersion of autoscaling rendered by this helm chart
  autoscaling: "" # autoscaling/v2
```

or 

```diff
-# -- Override APIVersions
+# Override APIVersions
apiVersionOverrides:
  # -- String to override apiVersion of autoscaling rendered by this helm chart
  autoscaling: "" # autoscaling/v2
```